src/render.spec.ts

```diff
@@ -0,0 +1,54 @@
+import expect from "expect";
+import { getRenderedGameState } from "./render";
+import { GameState, CellValue } from "./gamestate";
+
+function get2RowsGameStateAllCellsEmpty(): GameState {
+  return {
+    board: [
+      [{ value: CellValue.Empty }, { value: CellValue.Empty }],
+      [{ value: CellValue.Empty }, { value: CellValue.Empty }],
+    ],
+  };
+}
+
+function get2RowsGameStateAllCellsBlack(): GameState {
+  return {
+    board: [
+      [{ value: CellValue.Black }, { value: CellValue.Black }],
+      [{ value: CellValue.Black }, { value: CellValue.Black }],
+    ],
+  };
+}
+
+function get2RowsGameStateMixedCellsValues(): GameState {
+  return {
+    board: [
+      [{ value: CellValue.Empty }, { value: CellValue.Black }],
+      [{ value: CellValue.Empty }, { value: CellValue.Black }],
+    ],
+  };
+}
+
+describe("Get 2-rows board with all cells empty must return it", () => {
+  it("should pass", () => {
+    expect(getRenderedGameState(get2RowsGameStateAllCellsEmpty())).toBe(
+      "⬡⬡\n ⬡⬡"
```

Comment:
Not readable. Please use backticks to that the expected outputs looks like an actual board

Reply:
I'm not sure to understand how to do it with backticks. Could we check that tomorrow?